Nwp-soil-layers (#10)
MeteoSwiss/opendata-forecast-data: README.md

- [2.3 Accessing Forecast Data](#23-accessing-forecast-data)
- [2.4 3D Grid Structure and Representation](#24-3d-grid-structure-and-representation)
- [2.5 Accessing Static Grid Information: Height, Longitude, and Latitude](#25-accessing-static-grid-information-height-longitude-and-latitude)

> - [2.6 Data Visualisation with Jupyter Notebooks](#26-🚀-data-visualisation-with-jupyter-notebooks)

Reviewer: We're doing more than just visualizing (plotting) the data — we provide a full example that shows how to retrieve, process, and then plot the forecasts. The word "visualize" alone doesn't really capture the full scope of what the examples do. Here a suggestion
```suggestion
- [2.6 Example Notebooks: From Retrieval to Visualization](#26-🚀-example-notebooks-from-retrieval-to-visualization)
```

> - [2.6 Example Notebooks: From Retrieval to Visualization](#26-🚀-example-notebooks-from-retrieval-to-visualization)

Reviewer: The link didn't work with emoticones in the title - I suggest to remove the emoticone (As you prefer, or with emoticone but without link in the table of content
```suggestion
- [2.6 Example Notebooks: From Retrieval to Visualization](#26-example-notebooks-from-retrieval-to-visualization)
```

For more detailed information on the vertical grid, read section 3.4 in [Working with the ICON Model](https://www.dwd.de/DE/leistungen/nwv_icon_tutorial/pdf_einzelbaende/icon_tutorial2024.pdf?__blob=publicationFile&v=3).

> Besides the grid above the surface, there is also a grid below land. It is much simpler because the **level number corresponds to meters below ground**. For example, the parameter "soil temperature", abbreviated as T_SO, is constructed as describe above.

Reviewer: All correct, some reformulations suggestions.
```suggestion
In addition to the vertical grid above the surface, there is also a grid below the land surface. In this case, the **level numbers correpond directly to depths in meters below ground**. For example, the parameter "soil temperature" (abbreviated as `T_SO`) is defined using this subsurface vertical structure.
```

When opening a data set in a jupyter notebook the load function includes fetching the CLON/CLAT values. To retrieve CLON/CLAT without a python environment, see section 2.7.

> ### 2.6 🚀 Data Visualisation with Jupyter Notebooks

Reviewer: Re first comment:
```suggestion
### 2.6 🚀 Example Notebooks: From Retrieval to Visualization
```

> ### 2.6 🚀 Example Notebooks: From Retrieval to Visualization

Reviewer: Removing the emoticone in the title because we can't add the link to a title with emoticone in the table of contents at the beginning
```suggestion
### 2.6 Example Notebooks: From Retrieval to Visualization
```
